mac_client.md

```diff
@@ -0,0 +1,93 @@
+# Podman Mac CLient tutorial
+
+## What is the Podman Mac Client
+
+First and foremost, the Mac Client is under heavy development. We are working on getting the
+Mac client to be packaged and run for a native-like experience. This is the setup tutorial
+for the Mac client at its current stage of development and packaging.
+
+The purpose of the Podman for Mac is to allow users to run Podman on a Mac. Since Podman is a Linux
+container runtime, The Mac client is actually a version of the [remote-client](remote_client.md),
+edited to that the client side works on a Mac machine, and connects to a Podman "backend" on a Linux
+machine, virtual or physical. The goal is to have a naitive-like experience when working with the Mac
+client, so the command line interface of the remote client is exactly thesame as the regular Podman
+commands with the exception of some flags being removed as they do not apply to the Mac client.
+
+## What you need
+To use the Mac client, you will need a binary built for MacOS and a Podman "backend" on a Linux machine;
+hereafter referred to as the Podman node. In this context, a Podman node is a Linux system with Podman
+installed on it and the varlink service activated.  You will also need to be able to ssh into this
+system as a user with privileges to the varlink socket (more on this later).
+
+## Getting the Mac client
+The Mac client is availible through [Homebrew](https://brew.sh/).
+```
+$ brew cask install podman
+```
+
+## Setting up the client and Podman node connection
+
+To use the Mac client, you must perform some setup on both the your Mac and Podman nodes. In this case,
+the Mac node refers to the Mac on which Podman is being run; and the Podman node refers to where
+Podman and its storage reside.
+
+### Connection settings
+Your Linux box must have ssh enabled, and you must copt your Mac's public key from `~/.ssh/id.pub` to
+`/root/.ssh/authorized_keys` on your Linux box. This allows for the use of SSH keys for remote access.
+
+You may need to edit your `/etc/ssh/sshd_config` in your Linux machine as follows:
+```
+PermitRootLogin yes
+```
+
+Use of SSH keys are strongly encouraged over regular login. However, if you wish toavoid ‘logging in’ every
+time you run a podman command, you may edit your `/etc/ssh/sshd_config` on your Linux machine as follows:
+```
+PasswordAuthentication no
+PermitRootLogin without-password
+```
+
+### Podman node setup
+The Podman node must have Podman (not the Mac client) installed as normal. If your system uses systemd,
```

Comment:
suggest:
The Podman node must be running a Linux distribution that supports Podman and must have Podman (not the Mac client) installed.

Reply:
How would they know if their system uses systemd?

Reply:
Probably worth squeezing in a note somewhere that you'll need root access for the setup stuff on the Podman node.